primitives/benchmark/utils.go

```diff
@@ -0,0 +1,39 @@
+//go:build !benchmarks
```

Comment:
nit: unnecessary

Reply:
without it, the compiler complains that there is other declarations of the same functions